open-api/rest-catalog-open-api.yaml

```diff
@@ -2838,6 +2988,69 @@ components:
           additionalProperties:
             type: string
 
+    PreplanTableRequest:
+      type: object
+      required:
+        - select
+        - filter
```

Comment:
I don't think `filter` should be required here (it's also inconsistent with the PlanTableRequest)